### PR TITLE
Enable text autocorrect on Android with a flag to disable it

### PR DESCRIPTION
**Pull request by jason-simmons**
See https://github.com/flutter/flutter/issues/11168

### CODE REVIEW

**Comment by jason-simmons** [comment]
@cbracken

File: shell/platform/android/io/flutter/plugin/editing/TextInputPlugin.java [hunk 1]
```diff
@@ -77,7 +77,9 @@ public void onMethodCall(MethodCall call, Result result) {
         }
     }
 
-    private static int inputTypeFromTextInputType(String inputType, boolean obscureText) {
+    private static int inputTypeFromTextInputType(String inputType,
+                                                  boolean obscureText,
+                                                  boolean noAutoCorrect) {
```
Comment (cbracken) [inline shell/platform/android/io/flutter/plugin/editing/TextInputPlugin.java]: What about `bool autocorrect` to avoid the double-negative below?

Comment (jason-simmons) [inline shell/platform/android/io/flutter/plugin/editing/TextInputPlugin.java]: I'd like false to represent the default behavior and true to represent the special case (similar to obscureText)

Comment (Hixie) [inline shell/platform/android/io/flutter/plugin/editing/TextInputPlugin.java]: FWIW one of the lessons we learnt on the Web was that it's less confusing on the long run for the argument or property to be phrased in the positive than for "true" to be the default. We have several boolean arguments that default to true in the framework. (For example, we assiduously avoid having "hidden" and "disabled" arguments or properties.)

**Pull request merged**